src/modules/settings/settings.service.ts

```diff
@@ -0,0 +1,37 @@
+import { Injectable, Inject } from '@nestjs/common';
+import { SETTING_REPOSITORY } from 'src/core/constants';
+import { Setting } from './settings.entity';
+import { SettingDto } from './dto/setting.dto';
+import { Account } from '../accounts/accounts.entity';
+
+@Injectable()
+export class SettingsService {
+    constructor(@Inject(SETTING_REPOSITORY) private readonly settingRepository: typeof Setting) {}
+
+    async create(setting: SettingDto): Promise<Setting> {
+        return await this.settingRepository.create<Setting>({...setting})
+    }
+
+    async findAll(): Promise<Setting[]> {
+        return await this.settingRepository.findAll<Setting>({
+        	include: [{ model: Account, attributes: { exclude: ['password'] } }],
```

Comment:
lets exclude password by default. 
Sequelize provides an option to exclude properties by default. This will avoid repetition of code + exposing sensitive information by mistake.